ddl/db_test.go

```diff
@@ -3387,6 +3387,38 @@ func (s *testDBSuite2) TestDDLWithInvalidTableInfo(c *C) {
 	c.Assert(err.Error(), Equals, "[parser:1064]You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 94 near \"then (b / a) end));\" ")
 }
 
+// TestParallelTruncateTableAndAddColumn tests add column when truncate table.
+func (s *testDBSuite5) TestParallelTruncateTableAndAddColumn(c *C) {
```

Comment:
Could we put it to db_change_test.go? The parallel DDL tests are placed in this file.

Reply:
done.